### DISCUSSION TITLE
Help Needed: Connecting ABB Intercom to ESPBell-LITE

### DISCUSSION THREAD

**Discussion by jmttramos** [PricelessToolkit/ESPBell-LITE#23] [Q&A] [answered]
Hello everyone,

I'm working on a project to connect my ABB intercom to an ESPBell-LITE, but I've hit a roadblock and could really use some help. Below, I’ve attached photos of the intercom’s mainboard, the device itself, and the wall wires that I’m working with.

![20240824_105550_8752BBD5](https://github.com/user-attachments/assets/339fd1d2-ef66-40af-b3ec-5aff9544c0bc)
![20240824_144222_C474E4AA](https://github.com/user-attachments/assets/71f5474a-69c2-45f2-aec9-8ae3f39f63ed)
![20240824_144249_37D517D8](https://github.com/user-attachments/assets/2fbacd8b-aa8e-49ed-97df-0b68763bc4fd)
![20240824_144257_CEA11CBD](https://github.com/user-attachments/assets/c97c3663-f0d7-4cf4-b825-3b8bb76a59bf)

I want to integrate the ABB intercom with the ESPBell-LITE to enable remote doorbell notifications. However, I'm unsure how to correctly wire the intercom to the ESPBell-LITE and configure everything properly.

Can anyone guide me on how to correctly wire the ABB intercom to the ESPBell-LITE? Specifically, which wires should connect to which terminals on the mainboard and the ESP device?

Is there anything special I need to consider regarding the power supply when connecting these two devices, can I convert the power from the intercom to 3.3v using a DC2DC converter?

Any advice, wiring diagrams, or similar experiences would be greatly appreciated. Thanks in advance for your help!

Best regards,  
jmttramos

**Comment by PricelessToolkit** [comment, marked as answer]
Hi, your intercom is a digital , which unfortunately isn't compatible with ESPBell-LITE. Additionally, the buttons are capacitive rather than traditional pushbuttons, so direct connection to them isn't possible either.

**Reply by jmttramos** [reply]
Thanks a lot! And great project!!
